benchmarking/commons/benchmark_definitions/common.py

```diff
@@ -53,6 +55,7 @@ class SurrogateBenchmarkDefinition:
     max_resource_attr: Optional[str] = None
     datasets: Optional[List[str]] = None
     fidelities: Optional[List[int]] = None
+    points_to_evaluate: Optional[List[Dict[str, Any]]] = None
```

Comment:
Would you share more context regarding the new param?

Reply:
Here, you can list initial configurations, to be suggested by the scheduler first. For example, the Hugging Face benchmarks start from the defaults defined there.